lighthouse-core/lib/page-functions.js

```diff
@@ -225,6 +225,55 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
+ * This function checks if an element or an ancestor of an element is `position:fixed`.
+ * In addition we ensure that the element is capable of behaving as a `position:fixed`
+ * element, checking that it lives within a scrollable ancestor.
+ * @param {HTMLElement} element
+ * @return {boolean}
+ */
+/* istanbul ignore next */
+function isPositionFixed(element) {
+  /**
+   * @param {HTMLElement} element
+   * @return {boolean}
+   */
+  function hasScrollableAncestor(element) {
+    let currentEl = element.parentElement;
+    while (currentEl) {
+      if (currentEl.scrollHeight > currentEl.clientHeight) {
+        const yScroll = getStyleAttrValue(currentEl, 'overflowY');
```

Comment:
what's the circumstance that `yScroll` is falsy? isn't it always going to be set to one of the valid overflow values?

maybe I just need some help understanding the scrollable ancestor logic :) 

is it basically "the scroll height is greater than client height and the overflow isn't hidden"?

Reply:
Wow, this was a complete oversight. I've modified it to behave as expected.

> is it basically "the scroll height is greater than client height and the overflow isn't hidden"?

Yeah that's what we're looking for. Essentially saying that an element isn't truly fixed unless it behaves as fixed,ie . stays in place when the user scrolls.